app/build.gradle.kts

```diff
@@ -67,7 +69,15 @@ android {
         resValue("bool", "is_prerelease", "false")
 
         // Reads local.properties
-        val localProperties = gradleLocalProperties(rootDir)
+        val localPropertiesProvider = objects.fileProperty()
```

Comment:
Why switch to this when gradleLocalProperties exists??????

Reply:
it latest agp 8.3, it needs a direct File parameter intead of just rootDir, at least from what i have understood
otherwise fails sync and gives  this
/build.gradle.kts:82:45: No value passed for parameter 'providers'

its stub class hints this

```kotlin
public fun gradleLocalProperties(
    projectRootDir: File
): Properties
  com.android.build.gradle.internal.cxx.configure   GradleLocalPropertiesKt.class
  ```